sentry_sdk/integrations/django/__init__.py

```diff
@@ -646,8 +646,9 @@ def connect(self):
     ignore_logger("django.db.backends")
 
 
-# https://github.com/django/django/blob/6a0dc2176f4ebf907e124d433411e52bba39a28e/django/db/backends/base/base.py#L29
-# Avaliable in Django 1.8+
-def _set_db_system_on_span(span, vendor):
-    # type: (Span, str) -> None
+def _set_db_data(span, vendor, connection_params):
+    # type: (Span, str, Dict[str, str]) -> None
     span.set_data(SPANDATA.DB_SYSTEM, vendor)
+    span.set_data(SPANDATA.DB_NAME, connection_params.get("database"))
```

Comment:
Is the key called `database` for all db backends? Looks like for postgres it's `dbname`: https://github.com/django/django/blob/89c8c2e8311bd31b79c0829055d1e494ab88b801/django/db/backends/postgresql/base.py#L213 Or is this further normalized somewhere?

Reply:
[MySQL](https://github.com/django/django/blob/89c8c2e8311bd31b79c0829055d1e494ab88b801/django/db/backends/mysql/base.py#L217) seems to use `database`, but [Oracle](https://github.com/django/django/blob/89c8c2e8311bd31b79c0829055d1e494ab88b801/django/db/backends/oracle/base.py#L246C23-L246C23) seems to not surface the dbname in the params at all if I read the function right, just the [OPTIONS](https://docs.djangoproject.com/en/4.2/ref/databases/#id13).

Reply:
Good catch. (I tried it with Postgres and it worked, but will check for dbname too)

Reply:
Updated my code (also to only set those data items if we have something that is not none.)

other thing, oracle seems to only the the name of the db to find it: https://docs.djangoproject.com/en/4.2/ref/databases/#id13